ext/standard/assert.c

```diff
@@ -77,7 +77,7 @@ PHP_INI_BEGIN()
 	 STD_PHP_INI_BOOLEAN("assert.bail",		"0",	PHP_INI_ALL,	OnUpdateBool,		bail,	 			zend_assert_globals,		assert_globals)
 	 STD_PHP_INI_BOOLEAN("assert.warning",	"1",	PHP_INI_ALL,	OnUpdateBool,		warning, 			zend_assert_globals,		assert_globals)
```

Comment:
Shouldn't we also set warning=0?

Reply:
I think not. Exception takes priority over warning, so it doesn't issue a warning and then throw; it only throws. This way reverting to PHP < 8 behavior is only `assert.exception=0`.

Reply:
It seems pretty odd from a documentation perspective to have assert.warning=1 without actually enabling warnings. Ideally this would have been `assert.mode=exception` or so, because really only one of assert.bail, assert.exception and assert.warning makes sense at a time. But given things as they are, I would set it to zero.